glean-core/android/src/test/java/mozilla/telemetry/glean/scheduler/MetricsPingSchedulerTest.kt

```diff
@@ -272,8 +272,6 @@ class MetricsPingSchedulerTest {
 
             // Validate the received data.
             checkPingSchema(metricsJson)
-            assertEquals("The received ping must be a 'metrics' ping",
```

Comment:
Can we retain the check and use the submission URL here and at the other locations?

Reply:
adding it back